pgjdbc/src/main/java/org/postgresql/core/v3/QueryExecutorImpl.java

```diff
@@ -71,6 +71,7 @@
 public class QueryExecutorImpl extends QueryExecutorBase {
 
   private static final Logger LOGGER = Logger.getLogger(QueryExecutorImpl.class.getName());
+  private static final String COPY_ERROR_MESSAGE = "COPY commands are only supported using the CopyManager API.";
```

Comment:
That does not make the thing much cleaner. What do you think if we add a relevant URL to the documentation?

Reply:
While right now the documentation is pretty much static on URL definition, in the future there is a risk that a refactor to the documentation change the links...

And BTW I don't see the documentation for the CopyManager API in the index of the documentation.

Reply:
I don't think we should have direct URLs in the source code. Even with control of the host it's a pain if you ever decide to change things as there will be lingering references in the wild.

The error message should be enough that someone with a couple minutes of online searching could find. The first Google search result for "pgjdbc CopyManager" is the javadoc so I think that's good enough for now.

FYI, my last comment in the PR was referring to having *some* documentation about CopyManager in the docs. AFAICT there is nothing to which we could link in the docs even if we wanted to. At best it's be linking to a CopyManager example in the tests or the javadoc.